tutorials/securing-your-app-with-spring-security/04__form-less/content.adoc

```diff
@@ -0,0 +1,141 @@
+:title: Adding a page reload free login view for better UX using Spring Security
+:author: Paul Römer
+:type: text
+:tags: Spring, Spring Boot
+:description: Presents a Spring Security configuration that allows handling the whole authentication without submitting form data.
+:repo: https://github.com/vaadin-learning-center/spring-secured-vaadin/tree/form-less
+:linkattrs:
+:imagesdir: ./images
+
+After discussing different ways of creating login forms and POSTing the data to Spring Security's endpoint, now, we will explain how to *get authenticated without any page reload*. This approach will force us to dig a little deeper into the login flow Spring Security provides by triggering the authentication step manually and taking care of success and error handling on our own.
```

Comment:
Change "discussing" in this line to "exploring" - reason is "discussing" assumes two parties are speaking and really only you are speaking